tests/service_tests/consent_tests/test_consent_controller.py

```diff
@@ -109,7 +109,7 @@ def test_new_consent_validation(self):
 
         # Confirm a call to the dispatcher to rebuild the consent metrics resource data, with the ConsentFile.id
         # values from the expected_updates list
-        self.assertDispatchRebuildConsentMetricsCalled([2, 5, 6, 4])
+        self.assertDispatchRebuildConsentMetricsCalled([2, 5, 6, 4], call_count=2)
```

Comment:
if this is the only change in the unit test, is it expected behavior now for the dispatch task to be triggered twice?  What is happening on the extra call and what are the expected arguments?

Reply:
Yeah, it's expected to be called twice for the moment because the output strategy would call it twice (once each time it's `process_results` method is called, first time explicitly called to finalize any validations that were part of the first query and then again when the context ends).

In this case nothing is happening on the second call because all the expected consent files are handled by the first call. After being sure the ConsentResponse algorithm catches everytihng the old algorithm (being used as a fallback right now) will be taken out and the explicit `process_results` call won't be needed (so the call count will go back to 1 again).